packages/pharos/src/components/icon/pharos-icon.ts

```diff
@@ -30,13 +30,38 @@ export class PharosIcon extends PharosElement {
   /**
    * A description of what the icon represents
    * @attr description
+   * @deprecated Please use a11yLabel instead.
```

Comment:
This should be updated to "Please use a11yTitle instead."

```suggestion
   * @deprecated Please use a11yTitle instead.
```